.eslintrc.yaml

```diff
@@ -1,16 +1,29 @@
-extends: 'airbnb'
+env:
+  node: true
+
+extends:
+  - eslint:recommended
```

Comment:
The airbnb rules include baseline JavaScript rules that match their style guide, which is what we want. Combining both sets of rules will eventually lead to conflicts, and doesn't clearly improve anything. It's better in the long term for us to stick to one style guide, plus it's easier for developers to only reference one style guide when trying to figure out how to style code.